reana_resources_k8s/templates/config.yaml

```diff
@@ -0,0 +1,126 @@
+---
+# Deployment [local|prod]
```

Comment:
This file could be documented (via comments) just a little bit more and then included under "Configuration" section in the documentation. Shall we do it now or later?